x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.ts

```diff
@@ -218,7 +219,7 @@ export const CredentialsLogic = kea<CredentialsLogicType>({
     dataLoading: [
       () => [selectors.isCredentialsDetailsComplete, selectors.isCredentialsDataComplete],
       (isCredentialsDetailsComplete, isCredentialsDataComplete) => {
-        return isCredentialsDetailsComplete === false || isCredentialsDataComplete === false;
+        return isCredentialsDetailsComplete === false && isCredentialsDataComplete === false;
```

Comment:
I changed how this selector works for 2 reasons:

1. It allows me to use `isCredentialsDataComplete` as an individual `loading` indicator for the credentials table when paginating or deleting an item:
<img src="https://user-images.githubusercontent.com/549407/108774070-0fcfbb80-7514-11eb-9e10-e4d862c485e4.gif" alt="">

2. It still allows showing the "skeleton" loading state on initial page load and prevents a larger jump when all items load in, but doesn't re-show this state on simple pagination:
<img width="1127" alt="" src="https://user-images.githubusercontent.com/549407/108778077-b10d4080-7519-11eb-96c1-4af6bcd58bbc.png">